test/ck_tile/gemm_streamk/test_gemm_streamk.hpp

```diff
@@ -50,10 +51,10 @@ class TestCkTileStreamK : public ::testing::Test
               bool PadK       = true,
               bool Preshuffle = false,
               bool TransposeC = false>
-    bool invoke_streamk(const ck_tile::StreamKHostArgs& args,
-                        const ck_tile::stream_config& s,
-                        int num_cu,
-                        int occupancy)
+    std::tuple<bool, ck_tile::index_t> invoke_streamk(const ck_tile::StreamKHostArgs& args,
```

Comment:
Thinking we might avoid the return tuple by embedding the estimate_num_wgs_per_tile function into the TilePartitioner.